spock-core/src/main/java/org/spockframework/compiler/WhereBlockRewriter.java

```diff
@@ -16,21 +16,27 @@
 
 package org.spockframework.compiler;
 
-import java.util.*;
```

Comment:
I would gladly unify the styles and provide a config for `Idea` and `.editorconfig` and reformat the whole codebase to avoid this confusion (and so that new people have the correct settings from the start - it's not ok that everyone has to configure the styles on their own, and it's also not ok that the code is not uniform, and it's also not ok that I have to manually revert the imports and the trailing spaces)